### PR TITLE
Bump CSIDriver to v1

### PR DESCRIPTION
**Pull request by lizhuqi**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
CSIDriver v1beta1 seems to be deprecated. We shall bump to v1. 

I got error as below when I run the e2e test:
I0607 22:28:28.620449   14378 discovery.go:214] Invalidating discovery information
Error: unable to build kubernetes objects from release manifest: unable to recognize "": no matches for kind "CSIDriver" in version "storage.k8s.io/v1beta1"
helm.go:81: [debug] unable to recognize "": no matches for kind "CSIDriver" in version "storage.k8s.io/v1beta1"
unable to build kubernetes objects from release manifest

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

**Review by andyzhangx — COMMENTED** [review]
only need to change ` charts/latest/csi-driver-smb/templates/csi-smb-driver.yaml` since it would require minimum k8s version 1.18

**Comment by lizhuqi** [comment]
> only need to change ` charts/latest/csi-driver-smb/templates/csi-smb-driver.yaml` since it would require minimum k8s version 1.18

How about deploy/csi-smb-driver.yaml, charts/v1.0.0/csi-driver-smb/templates/csi-smb-driver.yaml and deploy/v1.0.0/csi-smb-driver.yaml ? Those are for latest and v1 right?

File: charts/v1.0.0/csi-driver-smb/templates/csi-smb-driver.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-apiVersion: storage.k8s.io/v1beta1
```
Comment (andyzhangx) [inline charts/v1.0.0/csi-driver-smb/templates/csi-smb-driver.yaml:2]: revert change here since v1.0.0 is released, only need to change `charts/latest/csi-driver-smb/templates/csi-smb-driver.yaml`

File: deploy/v1.0.0/csi-smb-driver.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-apiVersion: storage.k8s.io/v1beta1
```
Comment (andyzhangx) [inline deploy/v1.0.0/csi-smb-driver.yaml:2]: revert this change since v1.0.0 is already released.

File: charts/v1.0.0/csi-driver-smb/templates/csi-smb-driver.yaml (hunk 1 above)
Comment (lizhuqi) [reply charts/v1.0.0/csi-driver-smb/templates/csi-smb-driver.yaml]: reverted this change. Thanks for the code review!

**Review by andyzhangx — COMMENTED** [review]
/ok-to-test

**Review by andyzhangx — APPROVED** [review]
/lgtm
/approve
